client/test/build.gradle

```diff
@@ -30,6 +30,7 @@ install.enabled = false
 uploadArchives.enabled = false
 
 dependencies {
+  compile "org.apache.httpcomponents:httpcore:${versions.httpcore}"
```

Comment:
Yeah, I think it makes sense for that dependency to exist too. As it evolves, I suspect we'll add more and more helper methods similar to `ESTestCase`.